app/serializers/analytics_dashboard_organizer_serializer.rb

```diff
@@ -0,0 +1,124 @@
+class AnalyticsDashboardOrganizerSerializer < ActiveModel::Serializer
+  attributes :good_dreams, :dream_dates, :pie_chart, :stacked_bar, :scatter_plot
+
+  def good_dreams
+    object.dreams.map do |dream|
+      DreamSerializer.new(dream)
+    end
+  end
+
+  def dream_dates
+    dates = []
+    good_dreams.each do |dream|
+      dates << dream.nice_date
+    end
+    dates
+  end
+
+  def scatter_plot
+    data = { positive: [], negative: [], mixed: [],
+             positive_keywords: [], negative_keywords: [],
+             mixed_keywords: [] }
+    good_dreams.each do |dream|
+      dream.good_keywords.each do |keyword|
+        if keyword.positive?
+          data[:positive] << [keyword.relevance.to_f, keyword.sentiment.to_f]
+          data[:positive_keywords] << keyword.text
+        elsif keyword.negative?
+          data[:negative] << [keyword.relevance.to_f, keyword.sentiment.to_f]
+          data[:negative_keywords] << keyword.text
+        elsif keyword.mixed?
+          data[:mixed] << [keyword.relevance.to_f, keyword.sentiment.to_f]
+          data[:mixed_keywords] << keyword.text
+        end
+      end
+    end
+    data
+  end
+
+  def pie_chart
+    @_pie_chart_data ||= begin
+      [{
+            name: "Positive",
+            y: positive_result
+      }, {
+            name: "Negative",
+            y: negative_result,
+            sliced: true,
+            selected: true
+      }, {
+            name: "Neutral",
+            y: neutral_result
+      }, {
+            name: "Mixed",
+            y: mixed_result
+      }]
+    end
+  end
+
+  def stacked_bar
+    dates = dream_dates
+    keyword_data = [{
+                       name: 'Negative Mixed',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.